pyro/compressible/riemann.py

```diff
@@ -1,12 +1,15 @@
 import numpy as np
 from numba import njit
 
+import pyro.mesh.array_indexer as ai
+from pyro.util import msg
+
 
 @njit(cache=True)
-def riemann_cgf(idir, ng,
-                idens, ixmom, iymom, iener, irhoX, nspec,
-                lower_solid, upper_solid,
-                gamma, U_l, U_r):
+def riemann_cons(idir, ng,
```

Comment:
I think that this should still be named CGF, since this is the Colella, Glaz, Fergoson solver.  We will add more solvers and want to keep them straight.

Then you new wrapper can be used to get the flux as needed by calling the appropriate solver.

Reply:
now riemann_flux is the general interface to directly get flux from riemann solvers given direction. And riemann_cgf now gives conserved states.